### PR TITLE
vue.js complains fixes #107

### LINKED ISSUE #107: vue errors/warnings
## Abstract

I came across some errors/warnings in master while implementing #104 which should be dealt with in a seperate PR

```
[Vue warn]: Error compiling template:

value="[[amount]]": Interpolation inside attributes has been removed. Use v-bind or the colon shorthand instead. For example, instead of <div id="{{ val }}">, use <div :id="val">.

10 |  				<br><br>
11 |  				Amount:<br>
12 |  				<input type="number" name="amount" v-model="amount" value="[[amount]]" id="amount" v-bind:max="max_amount_sendable" min="0" step="1e-8" autocomplete="off">
   |                                                          ^^^^^^^^^^^^^^^^^^
13 |  				<div class="notification error" v-if="above_wallet_amount || needs_more_selected_coins">
14 |  					<ul>

(found in <Root>) vue.js:634:17

```

`[Vue warn]: Cannot find element: #spend_coins`

### PR DESCRIPTION
**Pull request by k9ert**
see #107

### CODE REVIEW

File: src/cryptoadvance/specter/templates/wallet_send.html [hunk 1]
```diff
@@ -57,15 +57,15 @@ <h1 class="padded">Sending to:</h1>
 			<div class="card"  id="spend_coins">
 				Recipient address:<br>
 				<div class="row">
-					<input type="text" id="address" name="address" v-model="address" value="[[address]]"> &nbsp;
+					<input type="text" id="address" name="address" v-model="address" :value="address"> &nbsp;
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/wallet_send.html:60]: I dont think you need :value="address" here, v-model does it already

File: src/cryptoadvance/specter/templates/wallet_send.html [hunk 2]
```diff
@@ -57,15 +57,15 @@ <h1 class="padded">Sending to:</h1>
...
 					<a class="btn" id="scanme"><img src="/static/img/qr_tiny.svg"/> Scan</a>
 				</div>
 				<br>
 				Address label (optional):<br>
 				<input type="text" id="label" name="label" value="{{label}}">
 				<br><br>
 				Amount:<br>
-				<input type="number" name="amount" v-model="amount" value="[[amount]]" id="amount" v-bind:max="max_amount_sendable" min=0 step="1e-8" autocomplete="off">
+				<input type="number" name="amount" v-model="amount" :value="amount" id="amount" v-bind:max="max_amount_sendable" min=0 step="1e-8" autocomplete="off">
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/wallet_send.html:68]: same as above, :value="amount" is not necessary

File: src/cryptoadvance/specter/__main__.py [hunk 3]
```diff
@@ -17,7 +17,6 @@
 from os import path
 import signal
 
-DEBUG = True
```
Comment (k9ert) [inline src/cryptoadvance/specter/__main__.py:20]: We will reestablish the default with this:
If you do a
FLASK_ENV=development
debug will be true by default.
If you want degub to be true in production, you can do:
FLASK_DEBUG=True

**Comment by stepansnigirev** [comment]
Looks good, merging!

**Pull request merged**